src/sentry/scim/endpoints/teams.py

```diff
@@ -0,0 +1,213 @@
+import logging
+import re
+
+from django.db import transaction
+from django.template.defaultfilters import slugify
+from rest_framework import serializers
+from rest_framework.exceptions import ParseError
+from rest_framework.response import Response
+
+from sentry.api.endpoints.organization_teams import OrganizationTeamsEndpoint
+from sentry.api.endpoints.team_details import TeamDetailsEndpoint, TeamSerializer
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.team import TeamSCIMSerializer
+from sentry.models import (
+    AuditLogEntryEvent,
+    OrganizationMember,
+    OrganizationMemberTeam,
+    Team,
+    TeamStatus,
+)
+from sentry.utils.cursors import SCIMCursor
+
+from .constants import SCIM_400_INVALID_FILTER, SCIM_404_USER_RES, GroupPatchOps
+from .utils import OrganizationSCIMTeamPermission, SCIMEndpoint, parse_filter_conditions
+
+delete_logger = logging.getLogger("sentry.deletions.api")
+
+
+CONFLICTING_SLUG_ERROR = "A team with this slug already exists."
+
+
+class SCIMTeamNameSerializer(serializers.Serializer):
```

Comment:
This class doesn't seemed to be used by anything. Is there some Django magic going on?

Reply:
good call, i think this got caught here from a git merge, will delete 👍🏼